unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PlayerInfoCardHUD/PlayerInfoCardHUDController.cs

```diff
@@ -110,17 +117,19 @@ private async UniTaskVoid CancelInvitationAsync()
         {
             try
             {
-                FriendRequest request = await friendsController.CancelRequestByUserIdAsync(currentPlayerId).Timeout(TimeSpan.FromSeconds(10));
+                friendOperationsCancellationTokenSource.Cancel();
```

Comment:
It's better to pass `CancellationToken` to the `async` flow from outside just like you did everywhere else. Please keep everything in alignment.